README.md

```diff
@@ -27,13 +27,35 @@ Swift Package Manager builds may also be made on Linux
 systems. Please see [DOCKER.md](DOCKER.md) and 
 [LINUX.md](LINUX.md) for details.
 
-## gRPC dependencies are vendored
+## CocoaPods integration
 
-Swift gRPC includes vendored copies of the gRPC core
-library and **BoringSSL**, an OpenSSL fork that is used by
-the gRPC Core. These are built automatically in Swift Package
+Swift gRPC is currently available [from CocoaPods](https://cocoapods.org/pods/SwiftGRPC).
+To integrate, add the following line to your `Podfile`:
+
+    pod 'SwiftGRPC'
+
+Then, run `pod install` from command line and use your project's generated
+`.xcworkspace` file.
+
+## Manual integration
+
+When not using CocoaPods, Swift gRPC includes **vendored copies** of the
+gRPC Core library and BoringSSL (an OpenSSL fork that is used by
+the gRPC Core). These are built automatically in Swift Package
 Manager builds.
 
+After [building your project](#building-your-project), add the generated
+`SwiftGRPC.xcodeproj` to your project, and add build dependencies
+on **BoringSSL**, **CgRPC**, **gRPC**, and **CzLib**.
```

Comment:
I think the zlib dependencies is obsolete, and `gRPC` is now called `SwiftGRPC`.

Reply:
Good catch (copy/paste). Updated here: https://github.com/grpc/grpc-swift/pull/216